src/unspread-lists.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import { visit, CONTINUE } from 'unist-util-visit';
+
+/**
+ * Collapse (un-spread) all lists
+ * @param {object} tree
+ */
+export function unspreadLists(tree, url, log) {
+  visit(tree, (node) => {
+    if (node.type === 'list' || node.type === 'listItem') {
+      // eslint-disable-next-line no-param-reassign
+      // node.spread = false;
+      if (node.spread) {
+        log.info(`Spread list seen in ${url}`);
+      }
+    }
+    if (node.type === 'listItem' && node.children.length > 1) {
+      // unwrap paragraphs if multiple children
+      // const children = [];
+      // node.children.forEach((child) => children.push(...child.children));
+      // eslint-disable-next-line no-param-reassign
+      // node.children = children;
+      log.info(`listItem with multiple children seen in ${url}`);
+    }
```

Comment:
I think it would be better to count them and only log onces.....

Reply:
That makes total sense 👍 

https://github.com/adobe/helix-html2md/pull/626/commits/a1fdc118628086efb54657ee7107ab6d92166a8b